Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -682,6 +682,30 @@ void MainWindow::loadSettings()
 
     if(connect_ok)
         QTimer::singleShot(0, this, &MainWindow::slotConnectToLatest);
+#if defined(Q_OS_WINDOWS)
+    bool SRActive = false;
+    Tolk_Load();
+    if(Tolk_DetectScreenReader() != NULL)
+        SRActive = true;
+    Tolk_Unload();
+    if(ttSettings->value(SETTINGS_GENERAL_FIRSTSTART, SETTINGS_GENERAL_FIRSTSTART_DEFAULT).toString() == "1")
+    {
+        QMessageBox answer;
+        answer.setText(tr("%1 has detected usage of screenreader on your computer. Do you whish to enable accessibility options offered by %1 with recommanded parameters?").arg(APPNAME_SHORT));
+        QAbstractButton *YesButton = answer.addButton(tr("&Yes"), QMessageBox::YesRole);
+        QAbstractButton *NoButton = answer.addButton(tr("&No"), QMessageBox::NoRole);
+        Q_UNUSED(NoButton);
+        answer.setIcon(QMessageBox::Question);
+        answer.setWindowTitle(APPNAME_SHORT);
+        answer.exec();
+
+        if(answer.clickedButton() == YesButton)
+        {
+            ttSettings->setValue(SETTINGS_TTS_ENGINE, 2);

```

Comment:
Don't use constants for values that have an corresponding enumeration.

I've corrected this.